src/lib/fetch/fetchComments.ts

```diff
@@ -0,0 +1,105 @@
+import { HttpStatusCode, IModify } from '@rocket.chat/apps-engine/definition/accessors';
+import { UIKitBlockInteractionContext } from '@rocket.chat/apps-engine/definition/uikit';
+import { LayoutBlock } from '@rocket.chat/ui-kit';
+
+import { TodoistApp } from '../../../TodoistApp';
+import { MiscEnum } from '../../enums/Misc';
+import {
+  getActionsBlock,
+  getButton,
+  getContextBlock,
+  getSectionBlock,
+} from '../../helpers/blockBuilder';
+import { getCommentsUrl } from '../../helpers/const';
+
+interface IComment {
+  id: string;
+  content: string;
+  posted_at: string;
+  attachment?: {
+    file_name: string;
+  };
+}
+
+export async function getComments(
```

Comment:
Put the call part inside a service and formatting can be here.
Also, I don't understand why the folders are called like that.
Separate stuff by domain not by name.
i.g. instead of having a folder called share
have a folder called comment and add comment stuff inside there